### DISCUSSION TITLE
Welcome to Clockify-PowerBI Discussions!

### DISCUSSION THREAD

**Discussion by OscarValerock** [OscarValerock/Clockify-PowerBI#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomhayw** [comment]
Hi there,
Do you have the individual M code for each table that the connector brings in to PowerBI?
Thanks

**Reply by OscarValerock** [reply]
@tomhayw, hi, please create this as a new discussion. https://github.com/OscarValerock/Clockify-PowerBI/discussions/new
